Source/Core/Core/PowerPC/Jit64Common/ConstantPool.cpp

```diff
@@ -59,5 +59,5 @@ Gen::OpArg ConstantPool::GetConstantOpArg(const void* value, size_t element_size
   _assert_msg_(DYNA_REC, info.m_size == value_size,
                "Constant has incorrect size in constant pool.");
   u8* location = static_cast<u8*>(info.m_location);
-  return Gen::M(location + element_size * index);
+  return location + element_size * index;
```

Comment:
Removal of `Gen::M` makes `#include "Common/x64Emitter.h"` irrelevant. Also remember to remove the forward declaration in the header file.